test/scale/nodevertical.go

```diff
@@ -0,0 +1,44 @@
+// Package openshift runs the OpenShift extended test suite.
+package openshift
+
+import (
+	"github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	kubev1 "k8s.io/api/core/v1"
+
+	"github.com/openshift/osde2e/pkg/helper"
+)
+
+var _ = ginkgo.Describe("Scaling", func() {
+	defer ginkgo.GinkgoRecover()
+	h := helper.New()
+
+	nodeVerticalTimeoutInSeconds := 3600
+	ginkgo.It("should be tested with NodeVertical", func() {
+		// setup runner
+		scaleCfg := scaleRunnerConfig{
+			Name:         "node-vertical",
+			PlaybookPath: "workloads/nodevertical.yml",
+		}
+		r := scaleCfg.Runner(h)
+
+		// only test on 3 nodes
+		r.PodSpec.Containers[0].Env = append(r.PodSpec.Containers[0].Env, kubev1.EnvVar{
+			Name:  "NODEVERTICAL_NODE_COUNT",
+			Value: "3",
```

Comment:
The max_pods count needs to be changed to 750 instead of the default ( 1000 pods ) in case we are setting the nodevertical node count to 3 to avoid scheduling issues.

Reply:
This actually isn't necessary as the script multiplies the vertical node count * 250 and uses that to set the max pods count in the script used in scale.go.